src/components/Footer.tsx

```diff
@@ -0,0 +1,108 @@
+import { useCurrentState, useTodosMethods } from '../context/Store';
+import { FilterField } from '../types/FilterField';
+import cn from 'classnames';
+import { deleteTodo } from '../api/todos';
+
+interface Props {
+  inputRef: React.RefObject<HTMLInputElement>;
+}
+
+export const Footer: React.FC<Props> = ({ inputRef }) => {
+  const { todos, filterField } = useCurrentState();
+  const { setFilterField, deleteTodoLocal, setTimeoutErrorMessage } =
+    useTodosMethods();
+
+  const { activeTodos: activeTodosAmount } = todos.reduce(
+    (acc, todo) => {
+      if (todo.completed) {
+        // eslint-disable-next-line no-param-reassign
+        acc.completedTodos += 1;
+      } else {
+        // eslint-disable-next-line no-param-reassign
+        acc.activeTodos += 1;
+      }
+
+      return acc;
+    },
+    { activeTodos: 0, completedTodos: 0 },
+  );
```

Comment:
why do you need completedTodos here, just filter the array and get the active todos length